README.md

```diff
@@ -176,9 +187,8 @@ $ sudo dnf install -y \
 
 # Clone repo and build
 $ git clone --recursive git@github.com:das-labor/neopg.git
-$ cd neopg/build
-$ cmake ..
-$ make
+$ cmake -Hneopg -Bneopg/build
```

Comment:
Didn't do that because -H/-B are undocumented, but after reviewing the discussion on the cmake mailing list, that's the stalemate result after some back and forth. Oh well. https://www.mail-archive.com/cmake-developers@cmake.org/msg16767.html